Minimal Web API/CustomRestrictParameter.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace DocumentEditorCore
+{
+    /// <summary>
+    /// This Class contains the property for Restrict editing
```

Comment:
Represents the parameters required for generating the hash while protecting and unprotecting a Word document.